website/docs/Installation/linux-install.md

```diff
@@ -1,64 +1,131 @@
 ---
 sidebar_position: 5
+title: Linux
+description: You can install Podman Desktop on Linux from Flathub, a Flatpak bundle, or a ZIP archive.
+tags: [podman-desktop, installing, linux, flathub, flatpak]
+keywords: [podman desktop, podman, containers, installing, installation, linux, flathub, flatpak]
 ---
 
-# Linux
+# Installing Podman Desktop on Linux
 
-Check out the [Downloads](/downloads/linux) section of this website to download either the [.flatpak](https://flatpak.org/setup/Fedora) file or the zip file depending on your preference.
+You can install Podman Desktop from:
 
-To use Podman Desktop, the latest version of Podman is required. Podman is a rootless, daemonless container engine. Read more about it [here](https://podman.io/whatis.html).
+* [Flathub](#flathub)
+* [A Flatpak bundle](#flatpak-bundle)
+* [An archive](#archive)
 
-## Installing Podman Desktop
+## Installing Podman Desktop from Flathub {#flathub}
 
-### 1. Using [FlatHub](https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop)
+Consider installing the Podman Desktop from Flathub to have:
 
-With FlatHub, you can install Podman Desktop with just a single command.
+* One command installation
+* Package updates
 
-> Follow the [setup guide](https://flatpak.org/setup/) before installing Podman Desktop
+#### Prerequisites
 
-```sh
-flatpak install --user flathub io.podman_desktop.PodmanDesktop
-```
-After installation, to open Podman Desktop, use the following command:
+* [Flatpak](https://flatpak.org/setup/)
+* [Podman](https://podman.io/whatis.html) stable version
 
-```sh
-flatpak run io.podman_desktop.PodmanDesktop
-```
+#### Procedure
 
-### 2. Using [Flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
+1. Assert the Flathub repository is enabled, and add it if required:
 
-:::infoPrerequisite 
-Verify Flatpak is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up Flatpak.
-:::
+    ```shell
+    $ flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
Having `$` on commands makes it difficult to use the "copy" functionality within Docusaurus to quickly copy-and-paste commands.

I remember `$` being *required* on OpenShift docs, but since we're upstream / community project, we don't require it, nor do we have `$` in other pieces of the docs. I think we shouldn't have this as `$` isn't very common on other community projects within Red Hat unless it's on the openshift docs site.

Reply:
I am afraid this is a rule that I will have to apply. It's not only OpenShift, but for all Red Hat documentations:

> If you include a shell prompt in a sample command, always show the correct prompt for a regular user ($) or a user with root privileges (#).

https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges

Reply:
For later: figure out a way for docusaurus to remove the $ from the copied content.